runtime/builtin/seq.ts

```diff
@@ -0,0 +1,127 @@
+import { SuIterable } from "../suvalue";
+import { maxargs } from "../su";
+import { add } from "../ops";
+import { cmp } from "../cmp";
+import { SuSequence, SequenceBase } from "./susequence";
+import { SuObject } from "../suobject";
+import * as util from "../utility";
+
+class SuSeq extends SuIterable {
+    private infinite: boolean = false;
+    private i: any;
+    constructor(private from: any, private to: any, private by: any) {
```

Comment:
Should "by" be saved in a member in SuSeq? Or maybe I'm missing something.

Reply:
"private by: any" in the parameter list declares that "by" is a private member and it will be initialized by the value passed to the constructor

Reply:
Ah, I see. I was thrown off by the explicit assignments to "to" and "from" but those are special cases.